CHANGES.txt

```diff
@@ -1,4 +1,5 @@
 Current
+Fixed: GITHUB-2301: Add support for object-based reporter configurations
```

Comment:
We usually encourage contributors to add their names as well to the PR so that it gives them the due credit. Can you please help add that ? Also I think this file has a conflict due to a recent merge of my PR and so you may have to rebase.

Reply:
DONE